apps/condo/domains/contact/components/ContactsEditor/index.tsx

```diff
@@ -331,52 +338,58 @@ export const ContactsEditor: React.FC<IContactEditorProps> = (props) => {
-                                                    onChecked={handleSyncedFieldsChecked}
-                                                    checked={editableFieldsChecked}
-                                                    contacts={fetchedContacts}
-                                                    displayMinusButton={true}
-                                                    onClickMinusButton={handleClickOnMinusButton}
-                                                    fields={fields}
-                                                    activeTab={activeTab}
-                                                    contactsLoading={contactsLoading}
-                                                    unitName={unitName}
-                                                />
-                                                {
-                                                    !breakpoints.TABLET_LARGE && (
-                                                        <Col span={24}>
-                                                            <Button
-                                                                type='link'
-                                                                style={BUTTON_STYLE}
-                                                                onClick={handleClickOnMinusButton}
-                                                                icon={<MinusCircleOutlined style={BUTTON_ICON_STYLE} />}
-                                                            >
-                                                                {CancelMessage}
-                                                            </Button>
-                                                        </Col>
-                                                    )
-                                                }
-                                            </>
-                                        ) : (
-                                            <Col span={24}>
-                                                <Button
-                                                    type='link'
-                                                    style={BUTTON_STYLE}
-                                                    onClick={handleClickOnPlusButton}
-                                                    icon={<PlusCircleOutlined style={BUTTON_ICON_STYLE}/>}
-                                                >
-                                                    {AddNewContactLabel}
-                                                </Button>
-                                            </Col>
-                                        )}
+                                        {(
+                                            displayEditableContactFields ||
+                                            (initialValue.id && !initialValueIsPresentedInFetchedContacts)) ||
+                                            initialValueWithoutContact
```

Comment:
Maybe convert this expression to some good called variable that describes what actually will be displayed